wger/gym/views/gym.py

```diff
@@ -124,10 +124,66 @@ def get_context_data(self, **kwargs):
         '''
         context = super(GymUserListView, self).get_context_data(**kwargs)
         context['gym'] = Gym.objects.get(pk=self.kwargs['pk'])
+        context['name'] = 'Active users'
         context['admin_count'] = len(context['object_list']['admins'])
         context['user_count'] = len(context['object_list']['members'])
-        context['user_table'] = {'keys': [_('ID'), _('Username'), _('Name'), _('Last activity')],
-                                 'users': context['object_list']['members']}
+        context['user_table'] = {'keys': [_('ID'), _('Username'), _('Name'), _('Last activity'), _('Status')],
+                                 'users': context['object_list']['members'],
+                                 "route": "active"}
+        return context
+
+class GymInactiveUserListView(LoginRequiredMixin, WgerMultiplePermissionRequiredMixin, ListView):
+    '''
+    Overview of all inactive users for a specific gym
+    '''
+    model = User
+    permission_required = ('gym.manage_gym', 'gym.gym_trainer', 'gym.manage_gyms')
+    template_name = 'gym/inactive_gym_users_list.html'
+
+    def dispatch(self, request, *args, **kwargs):
+        '''
+        Only managers and trainers for this gym can access the members
+        '''
+        if request.user.has_perm('gym.manage_gyms') \
+            or ((request.user.has_perm('gym.manage_gym')
+                or request.user.has_perm('gym.gym_trainer'))
+                and request.user.userprofile.gym_id == int(self.kwargs['pk'])):
+            return super(GymInactiveUserListView, self).dispatch(request, *args, **kwargs)
+        return HttpResponseForbidden()
+
+    def get_queryset(self):
+        '''
+        Return a list with the users, not really a queryset.
+        '''
+        out = {'admins': [],
+               'members': []}
+
+        for u in Gym.objects.get_members(self.kwargs['pk']).select_related('usercache'):
+            out['members'].append({'obj': u,
+                                   'last_log': u.usercache.last_activity})
+
+        # admins list
+        for u in Gym.objects.get_admins(self.kwargs['pk']):
+            out['admins'].append({'obj': u,
+                                  'perms': {'manage_gym': u.has_perm('gym.manage_gym'),
+                                            'manage_gyms': u.has_perm('gym.manage_gyms'),
+                                            'gym_trainer': u.has_perm('gym.gym_trainer'),
+                                            'any_admin': is_any_gym_admin(u)}
+                                  })
+        return out
+
+    def get_context_data(self, **kwargs):
+        '''
+        Pass other info to the template
+        '''
+        context = super(GymInactiveUserListView, self).get_context_data(**kwargs)
```

Comment:
***Code Quality: Nitpick***
Since you are using python 3. context can be copied over variable like so with a dictinary that is needed like so
``` 
{**context, 'gym': 'variable', 'name': 'variable'} and the likes
````